AirCasting/Utils/TimeConversion.swift

```diff
@@ -0,0 +1,22 @@
+// Created by Lunar on 09/09/2021.
+//
+
+import Foundation
+
```

Comment:
I think we shouldn't make those functions global but correct me if I'm wrong @pawelgil :) 
You can wrap those funcs into a TimeConverter class f.e. :)

Reply:
indeed. Please consider any global functions or variables as a no-go